controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/ProtoRole.java

```diff
@@ -0,0 +1,101 @@
+package com.yahoo.vespa.hosted.controller.api.role;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * This declares all tenant roles known to the controller. A role contains one or more {@link Policy}s which decide
+ * what actions a member of a role can perform.
+ *
+ * Optionally, some role definition also inherit all policies from a "lower ranking" role. Read the list of roles
+ * from {@code everyone} to {@code tenantAdmin}, in order, to see what policies these roles.
+ *
+ * See {@link Role} for roles bound to a context, where policies can be evaluated.
+ *
+ * @author mpolden
+ * @author jonmv
+ */
+public enum ProtoRole {
```

Comment:
How about `RoleDefinition`? That what's you call it in the doc anyway. 🙂

Reply:
Indeed.